village/src/components/SmurfForm.js

```diff
@@ -9,21 +10,28 @@ class SmurfForm extends Component {
       height: ''
     };
   }
+  
+  handleInputChange = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
 
   addSmurf = event => {
     event.preventDefault();
-    // add code to create the smurf using the api
 
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
+    const{name,age,height}=this.state;
+    const newSmurfs={name,age,height}
+
+    axios 
+    .post('http://localhost:3333/smurfs', newSmurfs)
+    .then(response => {
+      this.props.updateSmurfs(response.data);
```

Comment:
Careful with your method names. I would assume this is a PUT request with a name like updateSmurfs.